### ISSUE TITLE
Strange onPress effect Avatar

### ISSUE THREAD

**Issue by Fedour57** [react-native-elements/react-native-elements#2682]
**Explain what you did (Required)**

I'm trying to use an Avatar as button. The user can perform a click on the button.

**Expected behavior (Required)**

I expect a standard button press.

**Describe the bug (Required)**

When the user remains pressed on the avatar there is a strange shadow effect. The shape of a octogon appears. (Watch the screenshot)

This effect only appears if an elevation style is applied to the avatar.

**To Reproduce (Required)**

Create an Avatar with a elevation style

```
 <Avatar overlayContainerStyle={{
              height: 150,
              width: '100%',
              backgroundColor: 'white',
              elevation: 5,
              backgroundColor: '#7FC241'}} 
          size={80} 
          rounded 
          icon={{ name: 'check',color:"#FFF" }}
          onPress={() => this.confirmProduct()}
          activeOpacity={0.8}
          

/>
```

**Screenshots (Required)**

![20201116_204631](https://user-images.githubusercontent.com/74565493/99300561-12ec4900-284d-11eb-8e20-3d8ea9a9de09.jpg)

**Your Environment (Required):**

| software              | version |
| --------------------- | ------- |
| react-native-elements |    0.63.3     |
| react-native          |    1.2.7     |

**Comment by pranshuchittora** [comment]
Hi @Fedour57 I just tried to repro the issue on the Playground -> https://react-native-elements.js.org/#/avatar

![Screenshot from 2020-11-17 03-20-11](https://user-images.githubusercontent.com/32242596/99312415-07684480-2884-11eb-9c8b-d82a40b90973.png)


Unfortunately, I am not able to repro the issue. Can you prove more details about the platform (Android | iOS)

**Comment by Fedour57** [comment]
Hello, thanks you for your answer !

I'm using android. 


I have the same problem on all the android devices that I have tested. Also on emulator.
The problem does not appear on IOS

After some research, I realized that the problem also appears on FAB buttons which belongs to react-native-paper. I deduce that the problem does not come from react-native-elements but a problem with my app parameters.

I think we can close this issue !
